docs/_static/custom.css

```diff
@@ -95,6 +95,7 @@ div.admonition p {
 .admonition a {
     color: #eee;
     font-weight: 300;
+    text-decoration: underline dotted #eee;
```

Comment:
just curious: what is `#eee` for? Is it maybe the dot-size? or spacing?

Reply:
@Makman2 #eee is used for color in css :smile:

Reply:
I feel dumb now... seeing it two lines above 😅